RevenueCatUI/View+PurchaseRestoreCompleted.swift

```diff
@@ -53,6 +56,28 @@ extension View {
     /// [Documentation](https://rev.cat/paywalls)
     public func onPurchaseStarted(
         _ handler: @escaping PurchaseStartedHandler
+    ) -> some View {
+        return self.modifier(OnPurchaseStartedModifier(handler: { _ in
+            handler()
+        }))
+    }
```

Comment:
Should we mar this one as deprecated as well?

Reply:
Oh yes, good catch

Reply:
added in [f5c21b5](https://github.com/RevenueCat/purchases-ios/pull/3693/commits/f5c21b56a8baf64ea314e43f54f87908602d52fb) and [e5b935f](https://github.com/RevenueCat/purchases-ios/pull/3693/commits/e5b935fe84a7bf111dc838fc5a6a76d5cef7baf1)